tst/unit/test_meshblock_data_iterator.cpp

```diff
@@ -416,13 +428,13 @@ TEST_CASE("Coarse variable from meshblock_data for cell variable",
       REQUIRE(var.coarse_s.GetDim(2) == nside / 2 + 2 * nghost);
       REQUIRE(var.coarse_s.GetDim(1) == nside / 2 + 2 * nghost);
       AND_THEN("We can extract the fine object") {
-        auto pack = rc.PackVariables(std::vector<std::string>{"var"}, false);
+        auto pack = rc.PackVariables(std::vector<std::string>{"var"}).pack;
```

Comment:
```suggestion
        auto pack = mbd.PackVariables(std::vector<std::string>{"var"}).pack;
```

Reply:
I'll make this change manually, since removing the new packing interface changed this code.